core/src/rpc.rs

```diff
@@ -913,7 +912,7 @@ impl RpcSol for RpcSolImpl {
 
     fn get_version(&self, _: Self::Metadata) -> Result<RpcVersionInfo> {
         Ok(RpcVersionInfo {
-            solana_core: VERSION.to_string(),
+            solana_core: crate::version!().to_string(),
```

Comment:
[nits] [performance concern] This is Ok if [this new macro](https://github.com/solana-labs/solana/pull/6819/files#diff-b35d96f723d9e1ad4e01cbd9840b2754R9) is comparatively as cheap as [this previous `const` variable](https://github.com/solana-labs/solana/pull/6819/files#diff-59bdad60e705952dcb496cc6de3e34dfL1).

Or there is no concern whatsoever for the cost of this API endpoint.

Hopes `rustc` is smart enough, as I see the whole new macro is conceptually constantly evalatable.

Reply:
Good point.  Yes in this case I'm not concerned about the additional overhead (if any) to the getVersion end point